docs/source/release_notes.rst

```diff
@@ -14,6 +14,7 @@ Release Notes
         * Added more information to users about ensembling behavior in ``AutoMLSearch`` :pr:`1527`
         * Add woodwork support for more utility and graph methods :pr:`1544`
         * Changed ``DateTimeFeaturizer`` to encode features as int :pr:`1479`
+        * Return trained pipelines from ``AutoMLSearch.best_pipeline`` :pr:`1547`
```

Comment:
we should make sure to document this as a breaking change and let people know they can maintain current functionality by setting `train_best_pipeline=False`